ISISScientificComputing/autoreduce
# 377 serving djangodash in webapp

### Summary of work
* Changes to webapp test_settings.py to allow for use of django_plotly_dash framework
* Updates to Django webapp urls.py and run_summary.html template
* Creation of basic plot_handler script to show usage aided by README.md
* Minor alterations to plot_factory to return DjangoDash Dashapp instead of plain DashApp

### How to test your work
* Ensure that run summary page displays a plot when providing correct files in plot factory
* Ensure there is no drop in coverage

### Additional comments

Fixes #377 


**Before merging ensure the release notes have been updated**
